stablehlo/dialect/VhloTypes.td

```diff
@@ -228,6 +228,71 @@ def VHLO_UniformQuantizedV1 : VHLO_TypeDef<"UniformQuantizedV1", "quant_v1", "0.
   }];
   let assemblyFormat = "`<` $storageType `` `:` `` $expressedType `,` $scale `` `:` `` $zeroPoint `,` $storageTypeMin `` `:` `` $storageTypeMax `,` $flags `>`";
 }
+def VHLO_QuantizationScalesV1 : ArrayRefParameter<"::llvm::APFloat", "array of double scales"> {
+  let parser = [{
+    [&]() -> FailureOr<llvm::SmallVector<::llvm::APFloat>> {
+      ::llvm::SmallVector<double> scales;
+
+      auto parseResult = $_parser.parseCommaSeparatedList(AsmParser::Delimiter::Square, [&]() {
+        return $_parser.parseFloat(scales.emplace_back());
+      });
+      if(failed(parseResult)) return failure();
+      return llvm::to_vector(llvm::map_range(
+        scales, [](double scale) { return APFloat(scale); }));
+    }()
+  }];
+  let printer = [{
+    $_printer << '[';
+    llvm::interleaveComma($_self, $_printer, [&](APFloat scale) {
+      $_printer << scale;
+    });
+    $_printer << ']';
+  }];
+}
+def VHLO_QuantizationZeroPointsV1 : ArrayRefParameter<"int64_t", "array of int64_t zero points"> {
+  let parser = [{
+    [&]() -> FailureOr<llvm::SmallVector<int64_t>> {
+      ::llvm::SmallVector<int64_t> zeroPoints;
+
+      auto parseResult = $_parser.parseCommaSeparatedList(AsmParser::Delimiter::Square, [&]() {
+        return $_parser.parseInteger(zeroPoints.emplace_back());
+      });
+      if(failed(parseResult)) return failure();
+      return zeroPoints;
+    }()
+  }];
+  let printer = [{
+    $_printer << '[';
+    llvm::interleaveComma($_self, $_printer, [&](int64_t zeroPoint) {
+      $_printer << zeroPoint;
+    });
+    $_printer << ']';
+  }];
+}
+def VHLO_UniformQuantizedPerAxisV1 : VHLO_TypeDef<"UniformQuantizedPerAxisV1", "quant_per_axis_v1", "0.9.0", "current"> {
```

Comment:
I'm still debating if we should do this 1:1 with how we spec it, vs how it's implemented in the quant dialect. Let's discuss this more, i.e. should we instead introduce `VHLO_UniformQuantizedV2`.